src/main/java/com/rcv/ResultsWriter.java

```diff
@@ -261,26 +268,39 @@ private void generateSummarySpreadsheet(
       csvPrinter.println();
     }
 
-    // row for the exhausted CVR counts
-    // exhausted CVR header cell
-    csvPrinter.print("Exhausted ballots");
+    // row for the inactive CVR counts
+    // inactive CVR header cell
+    csvPrinter.print("Inactive ballots");
 
     // round indexes through all rounds
     for (int round = 1; round <= numRounds; round++) {
-      // count of votes exhausted this round
-      BigDecimal thisRoundExhausted = BigDecimal.ZERO;
+      // count of votes inactive this round
+      BigDecimal thisRoundInactive = BigDecimal.ZERO;
 
       if (round > 1) {
         // Exhausted count is the difference between the total votes in round 1 and the total votes
         // in the current round.
-        thisRoundExhausted =
-            totalActiveVotesFirstRound.subtract(totalActiveVotesPerRound.get(round));
+        thisRoundInactive =
+            totalActiveVotesFirstRound
+                .subtract(totalActiveVotesPerRound.get(round))
+                .subtract(roundToResidualSurplus.get(round));
       }
       // total votes cell
-      csvPrinter.print(thisRoundExhausted.toString());
+      csvPrinter.print(thisRoundInactive.toString());
     }
     csvPrinter.println();
 
+    // row for residual surplus (if needed)
+    Optional<BigDecimal> maxResidual =
+        roundToResidualSurplus.values().stream().max(BigDecimal::compareTo);
+    if (maxResidual.isPresent() && maxResidual.get().signum() == 1) {
```

Comment:
can you clarify what this line means?

Reply:
Just testing whether there's a positive value present. This is one of those cases where I know how to write it in an elegant way in Ruby, but in Java it's surprisingly complicated.

https://docs.oracle.com/javase/10/docs/api/java/math/BigDecimal.html#signum()

But upon further consideration... I can just test the final value, because the values increase monotonically. That will be simpler to read.